kubectl/open-notebook/client.go

```diff
@@ -107,11 +110,45 @@ func (c *notebookClient) portForward(ctx context.Context, verbose bool, nb *unst
 	} else {
 		stdout, stderr = io.Discard, io.Discard
 	}
-	fw, err := portforward.New(dialer, []string{fmt.Sprintf("%d:%d", localPort, podPort)}, ctx.Done(), ready, stdout, stderr)
-	if err != nil {
-		return err
+
+	// TODO: Remove hardcoding.
+	portPairs := []portPair{
+		{containerPort: 8888, localPort: 8888},
+		{containerPort: 8889, localPort: 8889},
```

Comment:
shouldn't the container port stay 8888?

Reply:
If we want to support serving both jupyter lab via the browser (8888) and a jupyter notebook server for an IDE to plug in, we need to forward another port (8889).

It turns out this pattern isn't vscode specific. [pycharm](https://www.jetbrains.com/help/pycharm/configuring-jupyter-notebook.html) also supports the same pattern.